CesiumGltfReader/include/CesiumGltfReader/GltfReader.h

```diff
@@ -210,7 +208,7 @@ class CESIUMGLTFREADER_API GltfReader {
   generateMipMaps(CesiumGltf::ImageCesium& image);
 
 private:
-  CesiumJsonReader::ExtensionReaderContext _context;
+  CesiumJsonReader::JsonReaderOptions _context;
```

Comment:
I noticed that `TilesetReader` contains these same changes. What are your thoughts on having them derive from a shared base class? (or really any way to avoid code duplication)

Reply:
I've moved to generating the reader classes in #711. GltfReader is still a bit special, though.

Reply:
I see it, TilesetReader.h looks to be deleted. Nevermind this comment then.